modules/ROOT/partials/commands/revisionhistory-cmds.adoc

```diff
@@ -0,0 +1,12 @@
+[cols="1,3",options="header"]
+|===
+|Command         |Description
+
+|revisionHistory   |Toggle the Revision History view
```

Comment:
```suggestion
|revisionHistory   |Toggle the {pluginname} view
```

Reply:
Cannot make this value dynamic as `modules/ROOT/pages/editor-command-identifiers.adoc` includes a ref to this file.
<img width="1158" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/40410898/1e3c05f3-84e2-48a1-aa70-279ff5df0be4">